src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -26,9 +30,17 @@ const SignedChatScreen = () => {
   } = useChatScreenHook(SIGNED);
 
   const flatlistRef = React.useRef<FlatList>();
+  const [loading, setLoading] = React.useState(false);
   const [, dispatchChannel] = (React.useContext(Context) as unknown as any).channel;
   const [profile] = (React.useContext(Context) as unknown as any).profile;
   const updatedChats = updateChatContinuity(chats);
+  const {signedProfileId} = useProfileHook();
+
+  const {moveToAnonymousChannel} = useMoveChatTypeHook();
+
+  const memberChat = selectedChannel?.rawJson?.channel?.members?.find(
+    (item: any) => item.user_id !== signedProfileId
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `any` type at line 34 and 35 is not recommended as it bypasses type checking. It's better to define a proper interface for the context. Also, error handling seems to be missing in case the `moveToAnonymousChannel` function fails. Consider adding some user feedback in case of an error.

```diff
- const [, dispatchChannel] = (React.useContext(Context) as unknown as any).channel;
- const [profile] = (React.useContext(Context) as unknown as any).profile;

+ interface IContext {
+   channel: [unknown, Dispatch<SetStateAction<unknown>>];
+   profile: [unknown, Dispatch<SetStateAction<unknown>>];
+ }
+ 
+ const { channel: [, dispatchChannel], profile: [profile] } = React.useContext<IContext>(Context);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->